tap_zendesk_chat/streams.py

```diff
@@ -116,6 +116,7 @@ def _bulk_chats(self, ctx, chat_ids: List):
         body = ctx.client.request(self.tap_stream_id, params=params)
         return list(body["docs"].values())
 
+    # pylint: disable=too-many-positional-arguments
```

Comment:
instead of ignoring pylints suggestion, let's just do away with positional arguments and require keyword arguments here

Reply:
you can do this by placing `*,` between `self,` and `ctx`.